src/common/eeprom.cpp

```diff
@@ -348,35 +346,40 @@ eeprom_init_status_t eeprom_init(void) {
     static eeprom_init_status_t status = EEPROM_INIT_Undefined;
     if (status != EEPROM_INIT_Undefined)
         return status; //already initialized
-    uint16_t version;
-    uint16_t features;
     status = EEPROM_INIT_Normal;
     osSemaphoreDef(eepromSema);
     eeprom_sema = osSemaphoreCreate(osSemaphore(eepromSema), 1);
     st25dv64k_init();
 
-    version = variant8_get_ui16(eeprom_get_var(EEVAR_VERSION));
-    features = (version >= 4) ? variant8_get_ui16(eeprom_get_var(EEVAR_FEATURES)) : 0;
-    if ((version >= EEPROM_FIRST_VERSION_CRC) && !eeprom_check_crc32())
+    //read entire eeprom at once
+    eeprom_vars_t &eevars = eeprom_startup_vars(); //initialize internal static variable, first call must be done now - to ensure interrupt is enabled
+
+    const uint16_t version = eevars.VERSION;
+    const uint16_t features = (version >= 4) ? eevars.FEATURES : 0;
```

Comment:
Can you name that magic number?

Reply:
it is not may code, I just simplified original expression
it is eeprom version 4 - (value stored in eeprom to know if eeprom should be updated) 
we had versions 2,4,6,7,8,9,10, I really don't know what happened with 1,3,5
I also do not understand why we need second variable features

Reply:
this code had been erased with commit removing version 2 support